injected/src/features/broker-protection/captcha-services/providers/hcaptcha.js

```diff
@@ -0,0 +1,109 @@
+import { PirError } from '../../types';
```

Comment:
Let's just null this one out like cloudflareTurnstile and not invoke it in the registry. I'd like for us to implement this as a follow-up to your initial work. Feel free to just roll to a separate PR and link in Asana.